test/registry/registry_tests.py

```diff
@@ -1372,39 +1372,6 @@ def test_cancel_upload(manifest_protocol, basic_images, liveserver_session, app_
     )
 
 
-def test_blob_caching(
```

Comment:
FTR: blob caching is already covered on https://github.com/quay/quay/blob/ba652e0b9879b319aabf26724a32acfbb7b74631/endpoints/v2/test/test_blob.py#L26